templates/flutter/lib/src/realtime_mixin.dart.twig

```diff
@@ -76,6 +76,23 @@ mixin RealtimeMixin {
             }
             break;
         }
+      }, onDone: () {
+        for (var list in _channels.values) {
+          for (var stream in list) {
+            stream.close();
+          }
+        }
+        _closeConnection();
```

Comment:
I think we should try to reconnect if there are channels or something similar to how we do for [web](https://github.com/appwrite/sdk-for-web/blob/master/src/client.ts#L257)

My server times out realtime connections after 2 minutes and using web, it reconnects automatically. Using the flutter SDK, it doesn't try to reconnect.